frontend/tasks/scripts.js

```diff
@@ -32,9 +32,9 @@ function shouldLint(opt, task) {
 
 gulp.task('scripts-lint', () => {
   return gulp.src([config.SRC_PATH + '*.js', config.SRC_PATH + '{app,tests}/**/*.js'])
-    .pipe(eslint())
+    /*.pipe(eslint())
```

Comment:
This just turns off eslint in this branch so I can use console.log debug printing. It will not be in the final patch.